app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -109,8 +113,9 @@ class MainActivity : AppCompatActivity() {
              var dailyActivityLog: DailyActivityLogDTO = gson.fromJson(jsonString, DailyActivityLogDTO::class.java)
 
              // using a CopyOnWriteArrayList prevents an error where more than one thread is operating on the list, leading to an error
-             val syncedActivityLog =  CopyOnWriteArrayList(dailyActivityLog.activityLog)
-             val syncedActivityNames = CopyOnWriteArrayList<String>();
+             val syncedActivityLog =
+                 CopyOnWriteArrayList(dailyActivityLog.activityLog)
```

Comment:
for properties with long initializers add a line break after '=' sign for better readability

https://kotlinlang.org/docs/reference/coding-conventions.html